yarn.lock

```diff
@@ -1945,7 +1945,7 @@ d@1:
   dependencies:
     es5-ext "^0.10.9"
 
-"dargs@github:christian-bromann/dargs":
+dargs@christian-bromann/dargs:
```

Comment:
This doesn't look intentional... not sure why it changed?

Reply:
Running `yarn install` on current master resulted in this update to the lockfile for me too. I tried deleting `node_modules` then `yarn install` again with this changed lockfile and things still build out properly

Reply:
ahh... common mistake. I ran into this recently, and it's just a weird API on their part.

`yarn install` pulls the latest dependencies which `package.json` allows. This might change your `yarn.lock`, if someone released a new version of a package recently.

`yarn install --pure-lockfile` installs the dependencies from `yarn.lock` without changing them.

Very unlikely to do any real harm, though.